ir/memory.cpp

```diff
@@ -1295,7 +1295,8 @@ void Memory::store(const Pointer &ptr,
 void Memory::storeLambda(const Pointer &ptr, const expr &offset,
                          const expr &bytes,
                          const vector<pair<unsigned, expr>> &data,
-                         const set<expr> &undef, uint64_t align) {
+                         const set<expr> &undef, uint64_t align,
+                         bool full_write) {
```

Comment:
Might be possible to avoid using this extra boolean, can consider this if correct direction.